content/en/blog/_posts/2022-04-15-grpc-probes-in-beta.md

```diff
@@ -0,0 +1,208 @@
+---
+layout: blog
+title: "Try the new gRPC probes in 1.24: now in beta"
+date: 2022-04-15
```

Comment:
Publish date should be after 2022-03-05 since this blog is a feature blog for the 1.24 release

Reply:
```suggestion
date: 2022-05-05
```